utils/entity.js

```diff
@@ -320,6 +199,193 @@ function prepareEntityForTemplates(entityWithConfig, generator) {
   return entityWithConfig;
 }
 
+function prepareEntityPrimaryKeyForTemplates(entityWithConfig, generator, enableCompositeId = true) {
```

Comment:
@mshima that is a very good idea for blueprints who don't want to support compositeIds.